release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -506,6 +506,21 @@ For more information, see  xref:../installing/install_config/configuring-firewal
 [id="ocp-release-notes-registry_{context}"]
 === Registry
 
+[discrete]
+[id="ocp-4-release-notes-read-only-registry-enhancements_{context}"]
+==== Read-only registry enhancements
+
+In previous versions of {product-title}, storage mounted as read-only returned no specific metrics or information about storage errors. This could result in silent failures of a registry when the storage backend was read-only. With this release, the following alerts have been added to return storage information when the backend is set to read-only:
+
+`ImageRegistryStorageReadOnly`::
+
+The image registry storage is read-only and no images will be committed to storage.
+
+`ImageRegistryStorageFull`::
+
+The image registry storage disk is full and no images will be committed to storage.
```

Comment:
As a nitpick, I suggest adding these commas because they are likely than not here as this is a two-clause sentence. Having these commas improves the readability of these sentences.

```suggestion
The image registry storage is read-only, and no images will be committed to storage.

`ImageRegistryStorageFull`::

The image registry storage disk is full, and no images will be committed to storage.
```

Reply:
Normally I would agree, how it's written is how it's returned in the alert: https://github.com/openshift/cluster-image-registry-operator/pull/1147/files#diff-e478597d655e139ab315adc42d9289776eebe8183facab70354d1687bbcd6018R25

Reply:
:joy: I am not surprised, seen this too many times myself. Apologies for bringing this up then!

Reply:
I never know whether to alter the text in our docs from the text that's written in, say, an alert. I always just mirror them, but sometimes I wish I wouldn't :D